shop/src/main/java/ro/msg/learning/shop/entity/OrderDetail.java

```diff
@@ -0,0 +1,28 @@
+package ro.msg.learning.shop.entity;
+
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import ro.msg.learning.shop.needed.OrderDetailId;
+
+import java.io.Serializable;
+
+@Entity
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Table(name="ORDER_DETAIL")
+public class OrderDetail implements Serializable {
```

Comment:
Why do you implement Serializable here? 
https://www.baeldung.com/jpa-entities-serializable